app/models/broadcast.rb

```diff
@@ -32,4 +34,12 @@ def single_active_announcement_broadcast
 
     errors.add(:base, "You can only have one active announcement broadcast")
   end
+
+  def update_last_active_at
```

Comment:
I think this might not be quite what you want. When I think `last_active_at` I think the last time the broadcast was active. Here is the scenario I am thinking that might be confusing. 

1. Set broadcast to active on March 1, last_active_at = March 1
2. Set broadcast to inactive on April 1, last_active_at does not change
3. View broadcast on May 1 and see `last_active_at` equals March 1 but you have no idea how long it was active for after it was marked active. In this case, `last_active_at` is actually `last_activated_at` If indeed this is what we want then I would suggest renaming the column to be more clear. 

If we actually want `last_active_at` then I would think you would want the value to be April 1 since that was the last time it was active. In order to set it this way you need to check if `active` has been changed from true to false and then set it when that condition occurs.

Reply:
You raise a good point @mstruve! The goal here is to display the time at which the Broadcast was last set to `active`: https://github.com/thepracticaldev/dev.to/issues/8276. That being said, I might want to change the column name to `last_actived_at` per your suggestion to clear the code up, unless I'm missing something here. 😊

Reply:
I think `last_activated_at` makes sense in the context of the issue, but I also see @mstruve's point about actually wanting to know _when_ it was last active. We don't use this feature yet, so it's hard to know whether we'll ever want to know when it was last _active_ versus when it was last _activated_.

It might be nice to actually track `last_active_at` and update it whenever someone toggles the `active` property. We don't technically need to know this, but since @mstruve pointed it out, it might be worth the extra few minutes of refactoring to make sure we're updating that column whenever we update `active` on the broadcast. 

Curious if others have strong opinions here (cc @mstruve + @atsmith813).

Reply:
Okay, I have modified this to actually track when the Broadcast was `last_active_at` and will push up the changes momentarily. Prior to doing that, I just wanted to make sure that the timestamps are granular enough as is and that I shouldn't change the timestamp format along with these changes -thoughts @mstruve @vaidehijoshi cc @atsmith813?

Reply:
@juliannatetreault are you asking if the column should no longer be a `datetime` format? if that's your question, then the answer is that you can keep it in the same format, since that is the convention (if you do a quick search through `schema.rb` you'll see that `created_at` and `updated_at` use that same format).

Reply:
No, what I'm asking is if the time displayed in `/internal/broadcasts` should be more granular (display it with the hour it was changed in addition to the date, for example) or if it is fine as is or if I should change the formatting of it in the view--sorry for the confusion! 😬  @vaidehijoshi

Reply:
@juliannatetreault sure, displaying the hour/minute seems specific enough 👍